src/stories/edit-collection/EditCollection.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { EditCollection } from '@/sections/edit-collection/EditCollection'
+import { WithI18next } from '../WithI18next'
+import { WithLayout } from '../WithLayout'
+import { WithLoggedInUser } from '../WithLoggedInUser'
+import { CollectionMockRepository } from '../collection/CollectionMockRepository'
+import { CollectionLoadingMockRepository } from '../collection/CollectionLoadingMockRepository'
+import { NoCollectionMockRepository } from '../collection/NoCollectionMockRepository'
+import { CollectionMother } from '../../../tests/component/collection/domain/models/CollectionMother'
+import { FakerHelper } from '../../../tests/component/shared/FakerHelper'
+import { MetadataBlockInfoMockRepository } from '../shared-mock-repositories/metadata-block-info/MetadataBlockInfoMockRepository'
+import { MetadataBlockInfoMockLoadingRepository } from '../shared-mock-repositories/metadata-block-info/MetadataBlockInfoMockLoadingRepository'
+import { MetadataBlockInfoMockErrorRepository } from '../shared-mock-repositories/metadata-block-info/MetadataBlockInfoMockErrorRepository'
+
+import { ROOT_COLLECTION_ALIAS } from '@tests/e2e-integration/shared/collection/ROOT_COLLECTION_ALIAS'
+
+const meta: Meta<typeof EditCollection> = {
+  title: 'Pages/Edit Collection',
+  component: EditCollection,
+  decorators: [WithI18next, WithLayout, WithLoggedInUser],
+  parameters: {
+    // Sets the delay for all stories.
+    chromatic: { delay: 15000, pauseAnimationAtEnd: true }
+  }
+}
+export default meta
+type Story = StoryObj<typeof EditCollection>
+
+export const Default: Story = {
```

Comment:
Since the Edit Page looks different depending on whether you are editing the Root collection, I think it would be good to have a story that shows both versions of the page.